### PR TITLE
Add latest round of missing boards

### PR DESCRIPTION
**Pull request by makermelissa**
This adds CircuitPython missing boards and Blinka missing boards.

### CODE REVIEW

File: _board/unexpectedmaker_feathers2_neo copy.md
Comment (tannewt) [inline _board/unexpectedmaker_feathers2_neo copy.md:1]: I suspect this file needs to be renamed so it doesn't include `copy`.

**Review by tannewt — CHANGES_REQUESTED** [review]
One weird filename. Looks good otherwise. Thanks for adding these!

File: _board/unexpectedmaker_feathers2_neo copy.md
Comment (makermelissa) [reply _board/unexpectedmaker_feathers2_neo copy.md:1]: Nice catch. I'll take a closer look.

Comment (makermelissa) [reply _board/unexpectedmaker_feathers2_neo copy.md:1]: Fixed.

**Review by dhalbert — APPROVED** [review]

**Pull request merged**